README.md

```diff
@@ -81,10 +80,10 @@ For each type of noun used in Authzed, there is a zed subcommand:
 For example, you can read Object Definitions in a Permissions System's Schema, check permissions, and even create or delete relationships.
 
 ```sh
-zed schema read user document
-zed permission check user:emilia writer document:firstdoc
-zed relationship create user:beatrice reader document:firstdoc
-zed relationship delete user:beatrice reader document:firstdoc
+zed schema read
+zed permission check document:firstdoc writer user:emilia
```

Comment:
Ideally, these should be checking permissions, so `read` and `write` (even though it works for both, of course)